modules/wsl-distro.nix

```diff
@@ -8,6 +8,16 @@ let
     exec ${pkgs.bashInteractive}/bin/sh "$@"
   '';
 
+  nixos-enter' = config.system.build.nixos-enter.overrideAttrs (_: {
```

Comment:
Do we already have the fix for https://github.com/NixOS/nix/issues/7602 ?

Reply:
It should be in the next 2.13 release.